### DISCUSSION TITLE
Show local logs without Google Cloud/BigQuery

### DISCUSSION THREAD

**Discussion by probablykasper** [supabase/supabase#15611] [Feature Requests]
[The documentation](https://supabase.com/docs/guides/getting-started/local-development#enabling-local-logging) says to configure Google BigQuery to see local logs, but that seems pretty excessive to me.

Perhaps Supabase Studio could show local logs without requiring BigQuery? Even if it supports less features, that would be nice. Alternatively, the Logs page could explain how to see the logs without BigQuery instead of just showing an error:

<img width="1272" alt="image" src="https://github.com/supabase/supabase/assets/11315492/df0c14bb-3a63-429b-9d5b-4daa2597ec31">

**Comment by Ziinc** [comment]
BigQuery is no longer necessary as the analytics server supports postgres as a backend.

Please see [updated local development instructions](https://supabase.com/docs/guides/cli/local-development#local-logging)
